components/ui/Window.js

```diff
@@ -42,19 +43,28 @@ const Title = styled.div`
   font-weight: normal;
   font-size: 32px;
   line-height: 41px;
-  padding: 16px;
+  padding: 13px 20px;
   background-color: ${props => props.theme.colors.vibrantCream};
   border-radius: 16px 16px 0 0;
   color: ${props => props.theme.colors.black};
   border-bottom: 2px solid ${props => props.theme.colors.vibrantBlack};
```

Comment:
Oh, and can the title be not selectable?

Reply:
yup, added that